config/v1/types_console.go

```diff
@@ -35,3 +36,20 @@ type ConsoleList struct {
 	metav1.ListMeta `json:"metadata,omitempty"`
 	Items           []Console `json:"items"`
 }
+
+type Auth struct {
+	// configmap reference in the openshift-config namespace
+	// The console needs a CA bundle to validate the certificate used to
+	// talk to the OAuth server.
+	OAuthEndpointCAFile ConfigMapReference `json:"oauthEndpointCAFile"`
+	// If authenticating with Request Header, OAuth or OpenID identity providers,
+	// an external URL is required to destroy single sign-on sessions.
+	LogoutRedirect string `json:"logoutRedirect"`
+}
+
+type Customization struct {
+	// TODO: How do we want to handle branding? Can we default if not provided?
+	Branding string `json:"branding"`
+	// TODO: How do we want to handle doc url? Can we default if not provided?
+	DocumentationBaseURL string `json:"documentationBaseURL"`
```

Comment:
We have to come up with a way to have reasonable defaults for these. I'm not sure what's best to key on (image name?)

We should add a type and constants for the possible values (`okd`, `ocp`, `online`, `dedicated`)

Reply:
Agree, I'll update.

Reply:
Per our discussion, I will drop the entire `Customization` struct.

Reply:
(This will instead exist on the console operator config)